content/messaging/customer-chat-api/v3.2/index.mdx

```diff
@@ -1155,6 +1155,89 @@ curl -X POST \
 
 </Section>
 
+<Section>
+<Text>
+
+### List Threads
+
+It returns threads that the current Agent has access to in a given chat.
+
+#### Specifics
+
+|                        |                                                                  |
+| ---------------------- | ---------------------------------------------------------------- |
+| **Method URL**         | `https://api.livechatinc.com/v3.2/agent/action/list_threads` |
+| **RTM API equivalent** | [`list_threads`](rtm-reference/#list-threads)            |
+| **Webhook**            | -                                                                |
+
+#### Request
+
+| Parameter          | Required | Data type | Notes                                                                                      |
+| ------------------ | -------- | --------- | ------------------------------------------------------------------------------------------ |
+| `chat_id`          | Yes      | `string`  |                                                                                            |
+| `sort_order` 		 | No       | `string`  | Possible values: `asc` - oldest threads first and `desc` - newest threads first (default). |
+| `limit`            | No       | `number`  | Default: 3, maximum: 100                                                                  |
+| `page_id`    		 | No       | `string`  | 		                                                                                     |
+| `min_events_count` | No       | `number`  | Range: 1-100; Specifies the minimum number of events to be returned in the response. It's the **total** number of events, so they can come from more than one thread. You'll get as many latest threads as needed to meet the `min_events_count` condition. |
+
+You cannot use both `limit` and `min_events_count` at the same time.
+
+</Text>
+<Code>
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/agent/action/list_threads \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+          "chat_id": "PWJ8Y4THAV",
+      }'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+    "threads": [{
+      "id": "K600PKZON8",
+      "active": true,
+      "user_ids": [
+        "b7eff798-f8df-4364-8059-649c35c9ed0c",
+        "smith@example.com"
```

Comment:
In customer-api we don't return emails as ID. @ale-kacperczyk I think this is consistent in other methods in docs, right? I have checked quickly `get_chat_threads` in docs and it has hashes instead of emails which is correct.

Reply:
That's correct, we're not using email addresses on the customer side. If we do, it's an overlooking. 
@kogierman, you can use this `id` for an agent: `bbb67d600796e9f277e360e842418833`